Resources/Prototypes/Entities/Objects/Weapons/Throwable/grenades.yml

```diff
@@ -46,6 +46,15 @@
     totalIntensity: 120 # about a ~4 tile radius
     canCreateVacuum: false
 
+- type: entity
+  name: phantom bang
```

Comment:
not only will you be able to see this on examine, but once you throw it people nearby are going to have no way of knowing they're about to get blown up. It's already hard enough to see a normal bomb in a crowded room, and making it invisible just makes it worse. This is effectively a murder method that's impossible to counter since you have no idea it's coming.

Reply:
ok, i added countdown sound for grenade